httpapi/router.go

```diff
@@ -49,6 +53,8 @@ func Router(
 	r.Route("/v1", func(r chi.Router) {
 		// core apis
 		coreapi.Register(r, nftSrv, configSrv, docsSrv, jobsSrv)
+		// user apis
+		userapi.Register(r, nftSrv, transferSrv)
```

Comment:
I am not really sure how this would look like when we want to have different versions, from core and userapi. Since the Router prefix `v1` for all services underneath. Lets keep it like this, but we might need to create either an explicit core route or duplicate the chi service.

Reply:
okay, I will make a note to rethink this.